contribution-guide.md

```diff
@@ -2,30 +2,30 @@
 
 Thanks for deciding to contribute.
 
-If you wish to send PR for adding articles or fixing issues, you need to first setup project locally. However, it is also important to understand our workflow so that your hard work is not wasted with rejected PR.
+If you wish to send PR for adding articles or fixing issues, you need to first setup project locally. However, it is also important to understand our workflow so that your hard work is not wasted in a rejected PR.
 
 ## Workflow
 
 We follow gitflow workflow.
 
-However please understand, starting September 2016, there will be no direct push to main repository `github.com/phpreboot/website`. All the changes will come through PR, regardless of change type. (Only kapilshamra have commit right but rule will apply on him as well)
+However, starting from September 2016, there will be no direct push to main repository `github.com/phpreboot/website`. All the changes will come through PR, regardless of change type. (Only kapilsharma have commit right but rule will apply on him as well)
 
-So first step of contribution is to fork the project on github. With fork, you will have different project like `github.com/<your-name>/website`. Now checkout with your fork.
+So first step of contribution is to fork the project on github, and then, you will have a different project like `github.com/<your-name>/website`. Now checkout it.
```

Comment:
Need to change `<your-name>` to `your-name`.

Reason, when GitHub convert MD to HTML, it ignores HTML tags & `<your-name>` is treated as HTML tag by GitHub. Thus in github, it looks like `github.com//website`.